js/script.js

```diff
@@ -0,0 +1,89 @@
+let canvas = document.getElementById('canvas');
+let jsonData2x2 = [];
+let jsonData32x32 = [];
+let image = new Image();
+
+fetch('./data/4x4.json')
```

Comment:
in future, when you'll be using Promises and imports, just import json files with 'import' or 'require' syntax. Also your files shouldn't be imported consistently (in your case the application **shouldn't** wait to import 32x32 and image.png files  until 4x4 file is loaded)